csrankings-d.csv

```diff
@@ -1149,7 +1150,9 @@ Dmitry Berenson,University of Michigan,https://web.eecs.umich.edu/~dmitryb,x-n9r
 Dmitry Chistikov,University of Warwick,https://warwick.ac.uk/fac/sci/dcs/people/dmitry_chistikov,NOSCHOLARPAGE
 Dmitry Evtyushkin,College of William and Mary,http://www.wm.edu/as/computerscience/faculty,QJ-pKL0AAAAJ
 Dmitry Goldgof,University of South Florida,http://aivi.cse.usf.edu/~goldgof,yQS_n0gAAAAJ
+Dmitry I. Ilvovsky,HSE University,https://www.hse.ru/staff/dilvovsky,n7VSUf8AAAAJ
 Dmitry Korkin,Worcester Polytechnic Institute,http://www.wpi.edu/academics/facultydir/dk2.html,V83az1AAAAAJ
+Dmitry P. Vetrov,HSE University,https://www.hse.ru/staff/dvetrov,7HU0UoUAAAAJ

```

Comment:
Title is research professor; only tenure-track faculty are eligible for inclusion.

Reply:
The Russian approach to job titles is weird. "Research professor" is more like an honorary title here. 
He is a tenured professor, teaching courses, advising PhD students (seven, as of now), and leading a research group.
But I can remove him from this PR if you insist.

Reply:
Sounds like he meets the inclusion criteria; thanks for the clarification.